### PR TITLE
Improves the consistency in capitalization of fetching and outputting of request headers

### PR DESCRIPTION
**Pull request by mhkuu**
Adapted during **Yoast Contributor Day** on Friday 4 November.

Improves the consistency in capitalization of fetching and outputting of request headers. See the Trac ticket for details.

Trac ticket: https://core.trac.wordpress.org/ticket/54225

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/Requests/Transport/fsockopen.php [hunk 1]
```diff
@@ -163,7 +163,7 @@ public function request($url, $headers = array(), $data = array(), $options = ar
 				$request_body = $data;
 			}
 
-			// Always include Content-length on POST requests to prevent
+			// Always include Content-Length on POST requests to prevent
```
Comment (costdev) [inline src/wp-includes/Requests/Transport/fsockopen.php]: This file is part of an external library and should not be patched in the PR.

File: src/wp-includes/SimplePie/Misc.php [hunk 2]
```diff
@@ -2146,7 +2146,7 @@ public static function output_javascript()
 		{
 			ob_start('ob_gzhandler');
 		}
-		header('Content-type: text/javascript; charset: UTF-8');
+		header('Content-Type: text/javascript; charset: UTF-8');
```
Comment (costdev) [inline src/wp-includes/SimplePie/Misc.php]: This file is part of an external library and should not be patched in this PR.

**Review by costdev — CHANGES_REQUESTED** [review]
Thanks for the PR @mhkuu! I've left a couple of notes about two files that are part of external libraries and should not be patched in this PR.

**Review by costdev — APPROVED** [review]
Thanks for the updates @mhkuu! LGTM :+1:

**Comment by audrasjb** [comment]
Committed in https://core.trac.wordpress.org/changeset/55210

**Pull request closed without merging**